pandas/_libs/tslibs/offsets.pyx

```diff
@@ -366,6 +366,22 @@ class ApplyTypeError(TypeError):
 cdef class BaseOffset:
     """
     Base class for DateOffset methods that are not overridden by subclasses.
+
+    Parameters:
+    -----------
+    n : int
+        Number of multiples of the frequency.
+
+    normalize : boolean
+        Whether the frequency can align with midnight.
+
+    Examples
+    --------
+    >>> pd.offsets.Hour(5).n
+    5
+
```

Comment:
```suggestion
```
If I remember correctly, blank lines in the examples cause two separate blocks to be rendered, which won't probably look very nice in this case.